CHANGELOG.md

```diff
@@ -1,47 +1,51 @@
 # Changelog
 
-## Unreleased
+## 4.5.0 (Feature release)
 
 ### New features
 
-#### Visually separate multiple summary lists on a single page with summary cards
+#### Use summary cards to visually separate multiple summary lists on a single page
 
-You can now use the [summary card](https://design-system.service.gov.uk/components/summary-list#summary-cards) to help you design and build pages with multiple summary lists to help you visually distinguish them and to allow users to use actions that apply to entire lists.
+You can now use the [summary card](https://design-system.service.gov.uk/components/summary-list#summary-cards). This new variant of the Summary list component can help you:
+
+design and build pages with multiple summary lists
+show visual dividers between summary lists
+allow users to apply actions to entire lists
```

Comment:
@owenatgov I think these need a line break between them?

The preview is squashing them all together (sorry!)

<img width="648" alt="Markdown example without new lines" src="https://user-images.githubusercontent.com/415517/214906061-2d6b494a-a4fc-44f5-aa9f-236df9ba6db5.png">

Reply:
Oof, missed that! Cheers.